localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/integration_response.py

```diff
@@ -110,19 +110,18 @@ def __call__(
                 headers[f"x-amzn-remapped-{header}"] = value
                 headers.remove(header)
 
-        # Those headers are passed through from the response headers, there might be more?
-        passthrough_headers = ("connection", "content-length")
+        # # Those headers are passed through from the response headers, there might be more?
```

Comment:
Can probably remove that ;)
```suggestion
        # Those headers are passed through from the response headers, there might be more?
```